Libraries/Text/Text.js

```diff
@@ -90,6 +91,10 @@ const Text = React.createClass({
      */
     onLongPress: React.PropTypes.func,
     /**
+     * Lets the user select text, to use the native copy and paste functionality.
```

Comment:
I believe you should add `@platform android` to the doc block along with `selectableAndroid` (kind of thing) so it's clear it's not supported on the iOS.